ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -1 +1,25 @@
 # local
+import ivy
+from ivy import with_supported_dtypes, to_ivy_arrays_and_back
+
+
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def instance_norm(x,
+                  running_mean,
+                  running_var,
+                  weight=None,
+                  bias=None,
+                  use_input_stats=True,
+                  momentum=0.9,
+                  eps=1e-05,
+                  data_format='NCHW',
+                  name=None):
+    return ivy.instance_norm(x,
```

Comment:
The signature of `ivy.instance_norm` only has `x, mean, variance, offset, scale, training, eps, momentum, data_format, out` arguments all these extra arguments that you're adding are causing failures when calling the function🙂

Reply:
I did try this parameters in the beginning. But the tests keeps failing with this error:
ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py:73 (test_paddle_instance_norm[cpu-ivy.functional.backends.numpy-False-False])
args = (dtype('O'),), kwargs = {}

Reply:
I can implement the instance_norm function in paddle backend to get rid of the  IvyNotImplementedException.
Can I do that ?